compair/static/modules/attachment/attachment-module.js

```diff
@@ -214,14 +214,17 @@ module.service('attachService',
             }
         };
 
-        uploader.onErrorItem = function(fileItem, response, status) {
+        uploader.onErrorItem = function(fileItem, response, status, headers) {
             fileItem.cancel();
             fileItem.remove();
             reset();
-            if (response == '413') {
+            if (status == 413) {
                 var upload_limit = UploadValidator.getAttachmentUploadLimit();
                 var limit_size = upload_limit / 1048576; // convert to MB
                 Toaster.error("File Not Uploaded", "The file is larger than the "+limit_size.toFixed(0)+"MB maximum. Please upload a smaller file instead.");
+            } else {
```

Comment:
Please add
```
if (...) {
...
} else if (response.title && response.message) {
     Toaster.error(response.title, response.message);
} else {
...
```
so that expected error messages can be shown